dot/network/service.go

```
@@ -415,17 +414,15 @@ func (s *Service) sentBlockIntervalTelemetry() {
 		}
 		finalizedHash := finalised.Hash()
 
-		err = telemetry.SendMessage(telemetry.NewBlockIntervalTM(
+		s.telemetry.SendMessage(telemetry.NewBlockIntervalTM(
```

Is it ok to ignore the error? :thinking:

---

Never mind, didn't see `SendMessage` no longer returns an error

---

`SendMessage` does not return error